docs/userguide/storagedriver/selectadriver.md

```diff
@@ -81,6 +81,12 @@ backing file system:
     |zfs            |Yes                           |
 
 
+**Note**
+Some storage drivers can not run over certain backing filesystem, details as below:
+
+1. overlay can not run over btrfs, aufs, zfs
+2. aufs can not run over btrfs, aufs
```

Comment:
I like the third column idea. Better than what I was thinking, which was a compat matrix.